.github/templates/Readme.md

```diff
@@ -4,9 +4,16 @@ We're using [ytt](https://github.com/vmware-tanzu/carvel-ytt) to generate the gi
 
 ## Prerequisites
 
+Get ytt from your package manager of choice. If, for example, you're on Windows and use chocolatey you can run:<br/> `choco install ytt -y`
+
+If instead ytt isn't in your packager manager or you don't use one you can manually get ytt as follows:
+
 1. Download the executable for your platform from the [releases page](https://github.com/vmware-tanzu/carvel-ytt/releases).
 1. Rename it to `ytt`.
 1. Place it in your PATH.
 
 ## Building the docs
-1. In `$SolutionDir/.github/templates` run `ytt -f main.yml --output-files ../workflows/`.
+1. `cd $SolutionDir/.github/templates`
+1. `ytt -f . --output-files ../workflows/` ==> to target all files<br/>
```

Comment:
In markdown we can insert a line break in 2 ways:
1. Specifically adding a line break tag (like a did)
2. Ending a line with 2 spaces and a new line char

I thought that in the option number 2 it would be harder to notice the new line. But maybe it's nicer to read? Which approach do you recommend?